.github/workflows/sycl_nightly.yml

```
@@ -27,6 +27,12 @@ jobs:
       lts_matrix: ${{ needs.test_matrix.outputs.lts_lx_matrix }}
       lts_aws_matrix: ${{ needs.test_matrix.outputs.lts_aws_matrix }}
 
+      # We use the build for the release, override its name and prefer
+      # widespread gzip compression.
+      artifact_archive_name: sycl_linux.tar.gz
```

```suggestion
      artifact_archive_name: dpcpp.lin.tgz
```
Technically the name of the compiler is oneAPI DPC++. What does `sycl` refer to? Branch name?
Mercy for my fingers, please! `.` is easier to type than `_`.

I think release version must be added to the file name, so we can keep multiple releases in the same directory.